responses/sk/HassTurnOff.yaml

```diff
@@ -4,6 +4,8 @@ responses:
     HassTurnOff:
       default: Turned off {{ slots.name }}
```

Comment:
Toto by mělo být přeloženo.

Reply:
pozri dalsi pull request :-)
https://github.com/home-assistant/intents/pull/731

Reply:
Takto to neprojde testem, takže stejně nelze potvrdit tento PR.

Dej oba do jednoho PR, ale nejdřív zkus projet testy, jestli vůbec má cenu žádat o PR.

Reply:
To je skoda, nemam rad miesat dve veci do jedneho, ale nieje to problem hodim to spolu vsetko teda, s opravenym single. Tento pull request teda mozme zavriet :-)